applications/cdr/src/cdr_util.erl

```diff
@@ -54,3 +59,16 @@ do_save_cdr(AccountMODb, Doc) ->
 -spec register_views() -> 'ok'.
 register_views() ->
     kz_datamgr:register_views_from_folder('cdr').
+
+-spec check_media_names(list()|kz_term:api_binary()) -> list().
+check_media_names(MediaNames) when is_list(MediaNames) ->
+    [HeadMN|TailMN] = MediaNames,
+    re:split(HeadMN, ",") ++ check_last_media_name(TailMN);
+check_media_names(MediaNames) ->
+    re:split(MediaNames, ",").
+
+-spec check_last_media_name(list())->list().
+check_last_media_name([FirstMediaName | []])->
```

Comment:
`[FirstMediaName | []]` is the same as `[FirstMediaName]` - the `| []` is implied.